mixer/cmd/init.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"strconv"
+
+	"github.com/clearlinux/mixer-tools/builder"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type initCmdFlags struct {
+	allLocal    bool
+	allUpstream bool
+	noDefaults  bool
+	clearVer    string
+	mixver      int
+	upstreamURL string
+	git         bool
+	format      string
+}
+
+var initFlags initCmdFlags
+
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize the mixer and workspace",
```

Comment:
> "Initialize the mixer and workspace",

This would make more sense saying just "Initialize the mixer workspace" or "initialize workspace and confs"
I know this is old & not your wording btw, but this is a great time to update it :)

Reply:
done